### PR TITLE
[ci skip] Fix docs for `default_form_builder`

### PR DESCRIPTION
**Pull request by Earlopain**
Passing a instance is not what you're supposed to do here. The instance will instead be passed to you in `form_with` and similar.

The top-level documentation already explains it correctly: https://api.rubyonrails.org/classes/ActionController/FormBuilder.html

### CODE REVIEW

File: actionmailer/lib/action_mailer/form_builder.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@ module ClassMethods
       # in the views rendered by this mailer and its subclasses.
       #
       # ==== Parameters
-      # * <tt>builder</tt> - Default form builder, an instance of ActionView::Helpers::FormBuilder
+      # * <tt>builder</tt> - Default form builder. Must be a subclass of ActionView::Helpers::FormBuilder
```
Comment (p8) [inline actionmailer/lib/action_mailer/form_builder.rb:26]: I don't think it needs to be a subclass, it just needs to have the same interface.
To keep inline with the documentation on line 7, maybe:
```suggestion
      # * <tt>builder</tt> - Default form builder. Accepts a subclass of ActionView::Helpers::FormBuilder
```

File: actionpack/lib/action_controller/form_builder.rb [hunk 2]
```diff
@@ -40,7 +40,7 @@ module ClassMethods
       # rendered by this controller and its subclasses.
       #
       # #### Parameters
-      # *   `builder` - Default form builder, an instance of
+      # *   `builder` - Default form builder. Must be a subclass of
```
Comment (p8) [inline actionpack/lib/action_controller/form_builder.rb:43]:
```suggestion
      # *   `builder` - Default form builder. Accepts a subclass of
```

File: actionmailer/lib/action_mailer/form_builder.rb (hunk 1 above)
Comment (Earlopain) [reply actionmailer/lib/action_mailer/form_builder.rb]: Yes, that's better. Thanks

**Comment by p8** [comment]
Thanks @Earlopain !

**Pull request merged**